fenix/app/src/main/java/org/mozilla/fenix/settings/logins/fragment/EditLoginFragment.kt

```diff
@@ -285,7 +304,7 @@ class EditLoginFragment : Fragment(R.layout.fragment_edit_login), MenuProvider {
         inflater.inflate(R.menu.login_save, menu)
     }
 
-    override fun onPrepareOptionsMenu(menu: Menu) {
+    override fun onPrepareMenu(menu: Menu) {
```

Comment:
nit: Changes not related to the issue should be avoid or at most be added to a separate commit.

Reply:
Tthe change is indeed related to the issue as the check mark doesn't update when `onPrepareOptionsMenu` is used.